deploy/ironic_bmo_configmap.yaml

```diff
@@ -4,10 +4,9 @@ metadata:
   name: ironic-bmo-configmap
 data:
   http_port: "6180"
-  http_ip: "172.22.0.1"
-  interface: "ens3"
+  interface: "eth2"
   dhcp_range: "172.22.0.10,172.22.0.100"
-  deploy_kernel_url: "http://172.22.0.1/images/ironic-python-agent.kernel"
-  deploy_ramdisk_url: "http://172.22.0.1/images/ironic-python-agent.initramfs"
-  ironic_endpoint: "http://localhost:6385/v1/"
-  ironic_inspector_endpoint: "http://localhost:5050/v1/"
+  deploy_kernel_url: "http://172.22.0.2/images/ironic-python-agent.kernel"
+  deploy_ramdisk_url: "http://172.22.0.2/images/ironic-python-agent.initramfs"
+  ironic_endpoint: "http://172.22.0.2:6385/v1/"
+  ironic_inspector_endpoint: "http://172.22.0.2:5050/v1/"
```

Comment:
What configures the provisioning nic on the cluster in this case?

Reply:
https://github.com/metal3-io/metal3-dev-env/pull/68/files#diff-f24cad65bbd2425d9a1a220a0dae09c8R77 adds the interfaces

https://github.com/metal3-io/metal3-dev-env/pull/68/files#diff-ee6766d0e2c6e5dfa4669f52720ee0c1R118 adds the IP

Reply:
Ah thanks, I missed that and wondered why we didn't need the static-ip-set container

Reply:
Ah, I didn't know about that. Is that way preferable? I could make it an init container on minikube.

Reply:
In this context it probably doesn't matter - in the multi-node cluster case you need some way to fail-over the provisioning IP between nodes, which the static-ip-set container does by continually refreshing a short lifetime on the ip configuration, that's probably not needed here so may as well stick with the simple ssh?